index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/main.css">
+  <link rel="stylesheet" href="./css/general.css">
+  <title>Rohit's Portfolio</title>
+</head>
+<body>
+  <header>
+    <div class="title">
+      <h5>Rohit Bhatt</h5>
+    </div>
+    <nav class="menu">
+     <a href="#">
+      <img class="hamburger-menu" src="./icons/hamburger-menu.png" alt="">
+     </a> 
+    </nav>
+  </header>
+  <main class="main-container">
```

Comment:
You're almost done, but you can follow the Figma design for improvements. Here is what you can do.

- [ ] 1. Please make sure to follow the exact colors, fonts, and sizes. So in here, it's highly recommended to update your background color, and resize your background image to match the Figma template.
- [ ] 2. The content for the `h1` tag looks great, but you can improve its design by setting its `line-height` to `64px`.
- [ ] 3. The gap between social media icons is not similar to the one in Figma design, so kindly make sure to update its styles.

![image](https://user-images.githubusercontent.com/34779928/198098792-34e7acab-1082-45a4-a94e-7f8ec1e2b732.png)

Reply:
updated